src/domain/landingPage/__tests__/LandingPagePreview.test.tsx

```diff
@@ -46,24 +56,35 @@ const renderComponent = () =>
     path: ROUTES.HOME_PREVIEW,
   });
 
-it('should render landing page preview correctly', async () => {
+it('should render landing page previews correctly', async () => {
   renderComponent();
 
   await waitFor(() => {
     expect(
-      screen.getByRole('heading', { name: landingPageTitle })
+      screen.getByRole('heading', { name: topBannerTitle })
     ).toBeInTheDocument();
   });
 
-  expect(screen.getByText(landingPageDescription)).toBeInTheDocument();
+  expect(screen.getByText(topBannerDescription)).toBeInTheDocument();
+  await waitFor(() => {
+    expect(
+      screen.getByRole('heading', { name: bottomBannerTitle })
+    ).toBeInTheDocument();
+  });
+  expect(screen.getByText(bottomBannerDescription)).toBeInTheDocument();
 });
 
-it('should show preview banner', async () => {
+it('should show preview banners', async () => {
   renderComponent();
 
   await waitFor(() => {
     expect(
-      screen.getByRole('heading', { name: landingPageTitle })
+      screen.getByRole('heading', { name: topBannerTitle })
+    ).toBeInTheDocument();
+  });
+  await waitFor(() => {
+    expect(
+      screen.getByRole('heading', { name: bottomBannerTitle })
     ).toBeInTheDocument();
   });
```

Comment:
`await screen.findByRole('heading', { name: bottomBannerTitle })`

would be better if we are waiting for an element

Reply:
that avoids use of waitFor

Reply:
fixed